pkg/wasm/imagefetcher.go

```diff
@@ -171,12 +172,12 @@ func extractDockerImage(img v1.Image) ([]byte, error) {
 		return nil, fmt.Errorf("could not fetch layers: %v", err)
 	}
 
-	// The image must be single-layered.
-	if len(layers) != 1 {
-		return nil, fmt.Errorf("number of layers must be 1 but got %d", len(layers))
+	// The image must have at least one layer.
```

Comment:
This change needs to be applied to `extractOCIStandardImage`, too, because the spec also mention about OCI standard image format.

Reply:
yeah true. https://github.com/istio/istio/blob/36a89eb800ec7aef49462329f283c5425f488f08/pkg/wasm/imagefetcher.go#L213-L216 these lines should  also be fixed. @jcchavezs

Reply:
Right. I forgot that one.